scripts/publish_crates.sh

```diff
@@ -56,13 +56,32 @@ set -x
 # though, so unless you have tools set up to run build scripts with the `--release` flag, we _should_ be fine,
 # but just in case:
 echo "MAKE SURE RUST ANALYZER, BACON, CARGO-WATCH etc are all OFF!"
-sudo pkill -9 rust-analyzer bacon cargo cargo-watch
+pkillexitstatus=0
+sudo pkill -9 rust-analyzer bacon cargo cargo-watch || pkillexitstatus=$?
+if [ $pkillexitstatus -eq 0 ]; then
+  echo "killed one or more processes"
+elif [ $pkillexitstatus -eq 1 ]; then
+  echo "no problematic processes found"
+elif [ $pkillexitstatus -eq 2 ]; then
+  echo "syntax error in the pkill command line"
+  exit $pkillexitstatus
+elif [ $pkillexitstatus -eq 3 ]; then
+  echo "fatal error"
+  exit $pkillexitstatus
+else
+  echo "unexpected error running pkill"
+  exit $pkillexitstatus
+fi
+
 
 rm -rf target_wasm # force clean build
 rm -f web_viewer/re_viewer_bg.wasm
 rm -f web_viewer/re_viewer.js
+rm -f web_viewer/re_viewer_debug_bg.wasm
+rm -f web_viewer/re_viewer_debug.js
 touch crates/re_viewer/src/lib.rs # force recompile of web server
 cargo r -p re_build_web_viewer -- --release
+cargo r -p re_build_web_viewer -- --debug
```

Comment:
Because of https://github.com/rerun-io/rerun/issues/1327 we now include both the release and debug .wasm in our published crates, which was not intended 🤦